db/schema.rb

```diff
@@ -11,23 +11,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-<<<<<<< HEAD
-ActiveRecord::Schema.define(version: 20161206021535) do
-=======
 ActiveRecord::Schema.define(version: 20161205164755) do
->>>>>>> master
+
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-<<<<<<< HEAD
-  create_table "games", force: true do |t|
```

Comment:
ok, thank you. Maybe someday I just might finish this minor task haha, sorry guys